src/lib/tls/tls13/tls_transcript_hash_13.cpp

```diff
@@ -169,8 +167,7 @@ void Transcript_Hash_State::update(std::span<const uint8_t> serialized_message_s
       if(serialized_message_length > 0 && *serialized_message == CLIENT_HELLO)
          {
          truncation_mark =
-            find_client_hello_truncation_mark({serialized_message,
-                                               serialized_message + serialized_message_length});
+            find_client_hello_truncation_mark(serialized_message_s);
```

Comment:
I noticed that you're using the `_s` suffix for `std::span<>`. Is that somewhat idiomatic? I don't mind it, but I also wouldn't want to enforce it.